Examples/UIComponentExample/Supporting Files/AppDelegate.swift

```diff
@@ -9,12 +9,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
-        TappableViewConfiguration.default = TappableViewConfiguration { view, isHighlighted in
-            let scale: CGFloat = isHighlighted ? 0.9 : 1.0
-            UIView.animate(withDuration: 0.2) {
-                view.transform = .identity.scaledBy(x: scale, y: scale)
-            }
-        }
+//        TappableViewConfiguration.default = TappableViewConfiguration { view, isHighlighted in
+//            let scale: CGFloat = isHighlighted ? 0.9 : 1.0
+//            UIView.animate(withDuration: 0.2) {
+//                view.transform = .identity.scaledBy(x: scale, y: scale)
+//            }
+//        }
```

Comment:
How come this has been commented out?

Reply:
It's just part of their example app and didn't wanna spend time making changes to make the example compile